README.md

```diff
@@ -0,0 +1,39 @@
+[![CI](https://github.com/bluesky/ophyd-async/actions/workflows/ci.yml/badge.svg)](https://github.com/bluesky/ophyd-async/actions/workflows/ci.yml)
+[![Coverage](https://codecov.io/gh/bluesky/ophyd-async/branch/main/graph/badge.svg)](https://codecov.io/gh/bluesky/ophyd-async)
+
+[![PyPI](https://img.shields.io/pypi/v/ophyd-async.svg)](https://pypi.org/project/ophyd-async)
+[![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+# ophyd_async
+
+Asynchronous Bluesky hardware abstraction code, compatible with control systems like EPICS and Tango
+
+This is where you should write a short paragraph that describes what your module does,
+how it does it, and why people should use it.
+
+<<<<<<< before updating
```

Comment:
Merge error?

Reply:
Ack removed this and the leftover boilerplate